requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ pandas>=1.5,<2
 matplotlib>=3.6.0,<4
 numpy>=1.23.3,<2
 opensearch-py>=2
-torch>=1.13.0
+torch<1.13.0
```

Comment:
Why we need a lower version?

Reply:
> In PyTorch before trunk/89695, torch.jit.annotations.parse_type_line can cause arbitrary code execution because eval is used unsafely.

What does it mean "before trunk/89695"?

Reply:
probably it's referring the torch repo before this merge https://github.com/pytorch/pytorch/commit/767f6aa49fe20a2766b9843d01e3b7f7793df6a3. after this merge, it trigger an unsafe issue in the new version and torch team is still working on it. 

It's new issue that happened after 11/26/2022 lately https://vuldb.com/?id.214417

Reply:
so the best thing we can do is avoid this new version until the issue is resolved by Pytorch team

Reply:
If we want to avoid this version let's put: `torch=1.12.0`.